utils/configuration_test.go

```diff
@@ -159,7 +159,6 @@ func TestParseBugsnagWithEnvironmentVariables(t *testing.T) {
 // If the storage backend is invalid or not provided, an error is returned.
 func TestParseInvalidStorageBackend(t *testing.T) {
 	invalids := []string{
-		`{"storage": {"backend": "postgres", "db_url": "1234"}}`,
```

Comment:
nit: can we keep this test case but with some other invalid backend name now that postgres will be valid?